src/devices/LidarLiteV3/samples/LidarLiteV3.Sample.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Device.I2c;
+using Iot.Device.DistanceSensor;
+
+namespace Iot.Device.DistanceSensor.LidarLiteV3Samples
+{
+    internal class Program
+    {
+        public static void Main(string[] args)
```

Comment:
We now have most of our samples being top-level programs, which really requires minimum number of changes here in order to have it like that. You don't have to do this but it would be good to have consistency across the repo. You can check any other sample in the repo for examples.

Reply:
Made the sample top-level, much more concise.